src/compiler/transformers/rewrite_aliased_paths.ts

```diff
@@ -0,0 +1,152 @@
+import { normalizePath } from '@utils';
+import { dirname, relative } from 'path';
+import ts from 'typescript';
+
+import { retrieveTsModifiers } from './transform-utils';
+
+/**
+ * Transform modules aliased with `paths` in `tsconfig.json` to relative
+ * imported in `.d.ts` files.
+ *
+ * @returns a TypeScript transformer factory
+ */
+export function rewriteAliasedDTSPaths(): ts.TransformerFactory<ts.Bundle | ts.SourceFile> {
```

Comment:
Just a thought, but what do you think about a function name like `rewriteAliasedDTSImportPaths()`?

Reply:
It's a bit on the long side, but I'm fine with it 👍